.github/workflows/doc-build.yml

```diff
@@ -43,6 +43,10 @@ jobs:
       cancel-in-progress: true
 
     steps:
+    - name: Apply container owner mismatch workaround
+      run: |
+        git config --global --add safe.directory ${GITHUB_WORKSPACE}
```

Comment:
this is not really needed since this specific step does not use containers but I guess it does not hurt either

Reply:
no screw that, I'll drop it